application/org.openjdk.jmc.flightrecorder.flameview-java/plugin.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2022, Datadog, Inc. All rights reserved.
```

Comment:
Needs to be an Oracle/Datadog dual copyright. Check other Datadog provided files for examples.